src/App.js

```diff
@@ -33,75 +26,17 @@ class App extends React.Component {
     this.balanceInputValueRef = React.createRef()
   }
   // ========= Insert Banknote Form ================
-  handleBalanceEnterClick = e => {
-    e.preventDefault()
-    if (this.state.legalPayload.includes(parseInt(this.state.balanceInputValue))) {
-      this.setState({
-        balance: parseInt(this.state.balanceInputValue) + parseInt(this.state.balance),
-        isRichest: this.state.products.every(
-          p => p.price <= parseInt(this.state.balance) + parseInt(this.state.balanceInputValue)
-        ),
-        errorMessageUnknowBanknote: '',
-      })
-    } else {
-      this.setState({
-        isError: true,
-        errorMessageUnknowBanknote: 'Unknow Banknote',
-      })
-    }
-  }
-  balanceInputValue = e => {
+  setValuesOfInsertBanknoteForm = balance => {
     this.setState({
-      balanceInputValue: parseInt(e.currentTarget.value),
+      balance: balance,
```

Comment:
You can use another way, with es6:
```
{
  prop: "value",
  balance,
}
```

Reply:
done
Link: #6
https://github.com/AlexandrVinogradov/vending-machine-react/blob/f2919431799bf73ed82a1ddbed6d3fc9ee8c981a/src/App.js#L31